### PR TITLE
Dex bootstrap

### PR DESCRIPTION
**Pull request by wangjj9219**

### CODE REVIEW

File: generate/generate.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ const generate = async () => {
   bytecodes.push(['DEX', ethers.utils.getAddress('0x0000000000000000000000000000000000000803'), dex]);
   deployedBytecodes.push(['DEX', ethers.utils.getAddress('0x0000000000000000000000000000000000000803'), dexDeployedBytecode]);
```
Comment (zjb0807) [inline generate/generate.js:70]: remove the `DEV`?

**Review by zjb0807 — COMMENTED** [review]
please fix the issue #139

File: contracts/dex/DEXV2.sol [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        address who,
+        address tokenA,
+        address tokenB
+    ) public view override returns (uint256, uint256) {
+        require(tokenA != address(0), "DEX: tokenA is zero address");
+        require(tokenB != address(0), "DEX: tokenB is zero address");
+
+        (bool success, bytes memory returnData) = PRECOMPILE.staticcall(
+            abi.encodeWithSignature(
+                "getProvisionPoolOf(address, address,address)",
```
Comment (zjb0807) [inline contracts/dex/DEXV2.sol:48]:
```suggestion
                "getProvisionPoolOf(address,address,address)",
```

**Comment by wangjj9219** [comment]
DEX has been deployed on Acala/Karura, but can be replaced with DEXV2's deployedBytecode via `evm.setCode`

**Review by zjb0807 — APPROVED** [review]
LGTM

**Pull request merged**